excel/src/main/java/tech/tablesaw/io/xlsx/XlsxReadOptions.java

```diff
@@ -12,8 +12,12 @@
 
 public class XlsxReadOptions extends ReadOptions {
 
+  /** Which table should be read? Null means no specific index was set. First table has index 0. */
```

Comment:
I think it sounds funny to put a question in the javadoc. How about "The table to read" instead of "Which table should be read?"